### ISSUE TITLE
Use latest Twilio SDK version to allow package to be installed using Composer v2.0

### ISSUE THREAD

**Issue by lptn** [aloha/laravel-twilio#175]
latest twilio/sdk 6.3.0. Current max version: 5.42.2

Note, twilio/sdk v5 has a problem reported by composer:

```
Deprecation Notice: Class Twilio\TwiML\Voice\Echo_ located in ./vendor/twilio/sdk/src/Twilio/TwiML/Voice/Echo.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v2.0. in phar:///usr/local/bin/composer/src/Composer/Autoload/ClassMapGenerator.php:201
```
_______
I also created an issue to update 5.x SDK branch to fix this issue on that branch: https://github.com/twilio/twilio-php/issues/628

**Comment by lptn** [comment]
@aloha 
pls check https://github.com/twilio/twilio-php/issues/616#issuecomment-617238040

can we migrate this package (next major version) to use twilio/sdk 6?

**Comment by coffe4u** [comment]
Looks like this hasn't been updated in over a year :(

This would be a big help. If I can find the time I'll try to get a PR together.

**Comment by Kokil** [comment]
Any update in this issue

**Comment by hannesvdvreken** [comment]
I've released [5.0.0-RC1](https://github.com/aloha/laravel-twilio/releases/tag/5.0.0-RC1) to upgrade this package. Probably I will create a RC2 with type hints too. But you can already checkout this one and see if it works for you.

You might need to add `"minimum-stability": "RC"` to your composer.json file to be able to test it out.
